lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -110,6 +110,8 @@ class Server {
       // see https://github.com/jshttp/mime-types/issues/66
       if (contentType) headers['Content-Type'] = mime.contentType(contentType);
 
+      headers['origin-agent-cluster'] = ['?1'];
```

Comment:
Should we put this behind our own url parameter like `isolated_frame` or something? This is non-default behavior that we only use for one test.

If not could you add this to L103.

Reply:
couple reasons why I didn't put this behind a query param:

- chrome doesn't like it when you mix responses from the same origin with and without this header. I found I had to reset chrome to undo this "mix responses" state when navigating to these pages directly via `yarn static-server`, which was pretty confusing
- At least for me, I always assumed iframing across different ports on localhost would be isolated. Probably best if just isolate by default for our smoke tests.